scripts/utils/referrals.ts

```diff
@@ -37,6 +38,7 @@ export async function fetchReferralEvents(
   protocol: Protocol,
   referrerIds?: Address[],
   useStaging = false,
+  endTimestamp?: Date,
```

Comment:
nit: since I've just merged the rename [PR](https://github.com/divvi-xyz/divvi-protocol-v0/pull/154).
```suggestion
  endTimestampExclusive?: Date,
```